website/templates/issue2.html

```diff
@@ -536,6 +610,70 @@ <h2 class="mb-2 text-xl font-bold leading-tight text-gray-900 ">{{ issue.descrip
             }
         });
     });
+    const openModal = (btc, eth, bch, userId, cve_score) => {
+        let severityLevel;
+        if (cve_score >= 9.0){
+            severityLevel = "critical";
+        } else if (cve_score >= 7.0) {
+            severityLevel = "high";
+        } else if (cve_score >= 4.0) {
+            severityLevel = "medium";
+        } else {
+            severityLevel = "low";
+        }
+
+        let suggestedTip;
+        if (severityLevel === "critical"){
+            suggestedTip = "$50";
+        } else if (severityLevel === "high") {
+            suggestedTip = "$20";
+        } else if (severityLevel === "medium") {
+            suggestedTip = "$10";
+        } else {
+            suggestedTip = "$5";
+        }
```

Comment:
It seems you can combine `severityLevel` and `suggestedTip` setting logic:
```suggestion
        let severityLevel;
        let suggestedTip;

        if (cve_score >= 9.0) {
            severityLevel = "critical";
            suggestedTip = "$50";
        } else if (cve_score >= 7.0) {
            severityLevel = "high";
            suggestedTip = "$20";
        } else if (cve_score >= 4.0) {
            severityLevel = "medium";
            suggestedTip = "$10";
        } else {
            severityLevel = "low";
            suggestedTip = "$5";
        }
```

Reply:
@arkid15r ohh I missed that , ThanksYou